local-storage/src/storage.rs

```diff
@@ -17,6 +17,9 @@ pub struct Storage {
     pub provider: Box<dyn StorageProvider>,
 }
 
+// TODO: Make this configurable
```

Comment:
This would be configured why what radios people plan on using to transfer this data in the future?

Reply:
I imagined that making this configurable would give people one more knob for tuning the data transfer parameters for their particular communication setup. In this case adjusting the block size could be used to create blocks which fit nicely inside of the system's MTU to maximize throughput/minimize fragmentation.